pimd/pim_cmd.c

```diff
@@ -6963,7 +6963,12 @@ DEFUN (ip_pim_joinprune_time,
        "Join Prune Send Interval\n"
        "Seconds\n")
 {
-	nb_cli_enqueue_change(vty, "/frr-pim:pim/join-prune-interval",
+	char jp_timer_xpath[XPATH_MAXLEN];
+
+	snprintf(jp_timer_xpath, sizeof(jp_timer_xpath),
+		 "/frr-pim:pim/address-family[address-family='%s']/join-prune-interval",
```

Comment:
Could we introduce a macro for `"/frr-pim:pim/address-family[address-family='%s']` and use it everywhere?

Reply:
Done